pkgs/development/haskell-modules/lib/compose.nix

```diff
@@ -463,4 +463,59 @@ rec {
   allowInconsistentDependencies = overrideCabal (drv: {
     allowInconsistentDependencies = true;
   });
+
+  # The motivation for this utility is for use for CI builds in order to avoid a
+  # full rebuild on every commit to the trunk development branch or every pull
+  # request.  For more details, see:
+  #
+  # https://harry.garrood.me/blog/easy-incremental-haskell-ci-builds-with-ghc-9.4/
+  #
+  # This accelerates a Haskell package build by building the package
+  # "incrementally", meaning that a "full" rebuild is only done once every
+  # interval and all rebuilds in between are "incremental", meaning that each
+  # incremental build reuses the `dist` directory from the last full rebuild.
+  #
+  # This only works for packages that use `git` for their source.
+  incremental = { interval }: pkg:
```

Comment:
Could you give one or two examples of what sorts of values the `interval` argument should take in the above comment?